### PR TITLE
igb: fix build on 32-bit kernels

### PR DESCRIPTION
**Pull request by htfujina**
Fix build on 32-bit kernels.

### CODE REVIEW

**Comment by andrew-elder** [comment]
There are some extra white space changes in the commit. Are they deliberate?

**Comment by andrew-elder** [comment]
Can someone at Intel confirm/approve?

**Comment by intel-ethernet** [comment]
Technical changes look good to me. I’ll let Todd comment on the whitespace.

**Comment by htfujina** [comment]
The spacing fixes are to conform with the kernel standard (as I understand it). Not necessary, but the igb driver that this is based off of has those now as well.

**Comment by andrew-elder** [comment]
Ok, if spaces come from somewhere upstream and are not "accidental" I'll accept the patch as is.

File: kmod/igb/igb_ptp.c [hunk 1]
```diff
@@ -369,17 +365,14 @@ static int igb_ptp_gettime64_i210(struct ptp_clock_info *ptp,
 
 #ifdef HAVE_PTP_CLOCK_INFO_GETTIME64
 static int igb_ptp_settime64_82576(struct ptp_clock_info *ptp,
-				 const struct timespec *ts64)
+				   const struct timespec *ts64)
```
Comment (MarcinMiklasSTC) [inline kmod/igb/igb_ptp.c]: Why the parameter type wasn't changed here as in other functions? Shouldn't we have `const struct timespec64 *ts64` here?

Comment (htfujina) [inline kmod/igb/igb_ptp.c]: I think you're correct.

Comment (andrew-elder) [inline kmod/igb/igb_ptp.c]: @htfujina - will there be a new pull request to fix this then?

Comment (htfujina) [inline kmod/igb/igb_ptp.c]: Eventually, yes, but the history of timespec64 makes me think this won’t cause a failure and unless it does it’s not my highest priority. If nothing awful happens in the next week, I’ll have a pull request in a few days.

Todd Fujinaka
Software Application Engineer
Networking Division (ND)
Intel Corporation
todd.fujinaka@intel.com
(503) 712-4565

**Pull request merged**